src/lib/seam/components/DeviceDetails/DeviceDetails.tsx

```diff
@@ -35,148 +22,15 @@ export function DeviceDetails({
     return null
   }
 
-  if (!isLockDevice(device)) {
-    return null
-  }
-
-  return (
-    <LockDeviceDetails className={className} device={device} onBack={onBack} />
-  )
-}
-
-function LockDeviceDetails(props: {
```

Comment:
This was moved unchanged to to `src/lib/seam/components/DeviceDetails/LockDeviceDetails.tsx`